app/client/components/panel.es6

```diff
@@ -17,8 +17,14 @@ export const Heading = (props) => {
 
   return (
     <div className="panel-heading">
-      {panel_title}
-      {props.children}
+      <div className="row">
```

Comment:
This moves the "What's this?" tooltip on the advanced set shaper into the middle of the header. Adding pull-right as a class for the second column should fix it. The float: right property on the delete button can then be removed too.